include/openspace/scene/scenegraphnode.h

```diff
@@ -181,6 +181,7 @@ class SceneGraphNode : public properties::PropertyOwner {
     properties::StringProperty _guiDisplayName;
     properties::StringProperty _guiDescription;
     properties::BoolProperty _useGuiOrdering;
+    properties::BoolProperty _guiFocussable;
```

Comment:
sanity check - but surely, it should be "focusable" with one s, right?  That's how English works? 🤔 :)

Reply:
Yea.. after being awake for about 30h, my English proficiency goes down the drain ;)